Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment.yml

```diff
@@ -165,3 +166,4 @@ inputs: []
 outputs: []
 tests:
 - No tests - Deprecated playbook
```

Comment:
@tomer-pan 
Keep the same format
tests:
- no test

Reply:
@David-BMS I don't think that this is something that I changed - it was there already. Also, there are a lot of different formats across all playbooks. I can change this one though, no problem. I do think that it's not worth the time to change other deprecated playbooks to match this format.